index.html

```diff
@@ -1,29 +1,183 @@
 <!DOCTYPE html>
-
 <html lang="en">
 
 <head>
-    <title>
-        Youtube Clone
-    </title>
-    <link href="fonts/fontawesome-free-5.14.0-web/css/all.css" rel="stylesheet">
-    <link href="css/style.css" rel="stylesheet">
-
+  <link rel="icon" type="image" href="images/youtube.ico" />
+  <title>Youtube-clone: Microverse Projects</title>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="css/style.css">
+  <link rel="stylesheet" href="fonts/fontawesome-free-5.14.0-web/css/all.css">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Font+Name">
 </head>
 
 <body>
-    <nav class="navbar">
-        <ul class="main">
-        <li><a href=""><i class="fas fa-bars"></i></a></li>
-        <li><a class="logo" href=""><img src="images/icons8-play-button-96.png" alt="" height="40px" width="40px">Youtube</a></li>
-        <li><form action="">
-            <input type="search" name="" id="" placeholder="Search">
-            <button><i class="fa fa-search"></i></button>
-        </form></li>
-        <li><a href=""><i class="fa fa-upload" ></i></a></li>
-        <li><button>Sign in</button></li>
-    </ul>
-    </nav>
+  <nav class="navbar">
+    <div class="navbar-items">
+      <i id="menu-icon" class="material-icons">menu</i>
+      <img class="logo" src="images/youtube.png">
+    </div>
+    <div class="navbar-items">
+      <div class="wrap">
+        <div class="search">
+          <input type="text" class="searchTerm" placeholder="Search">
+          <button type="submit" class="searchButton">
+            <i class="fa fa-search"></i>
+          </button>
+        </div>
+      </div>
+    </div>
+    <div class="navbar-items">
+      <div class="notification">
+        <i id="upload-icon" class="fas fa-upload"></i>
+      </div>
+      <div class="notification">
+        <i id="apps-icon" class="material-icons">apps</i>
+      </div>
+      <div class="notification">
+        <i id="notification-icon" class="fas fa-bell"></i>
+      </div>
+      <div class="notification">
+        <i id="user-icon" class="far fa-user-circle"></i>
+      </div>
+    </div>
+  </nav>
+  <section class="container">
+    <section class="video">
+        <iframe width="560" height="315" src="https://www.youtube.com/embed/V74l_zS1x8E" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>            
+        <section class="description">
+          <section class="video-details">
```

Comment:
- [ ] **OPTIONAL** Try to use the `<main>` tag for indicating the dominant content of the `<body>` of a document